guides/v2.3/graphql/queries/category.md

```diff
@@ -242,16 +178,87 @@ The following query returns breadcrumb information about the women's tops catego
+          "category_url_key": "tops-women",
+          "category_url_path": "women/tops-women"
         }
       ]
     }
   }
 }
 ```
+
+## Input attributes
+
+Attribute | Data type | Description
+--- | --- | ---
+`id` | Int | The category ID to use as the starting point of your category search.
+
+## Output attributes {#Categories}
+
+The query returns a `CategoryTree` object, which implements `CategoryInterface`. It can contain the following top-level attributes:
+
+Attribute | Data type | Description
+--- | --- | ---
+`breadcrumbs` | `Breadcrumb` | A `Breadcrumb` object contains information the categories that comprise the breadcrumb trail for the specified category
+`children` | `CategoryTree` | A `CategoryTree` object that contains information about a child category. You can specify up to 10 levels of child categories.
+`created_at`| String | Timestamp indicating when the category was created
+`default_sort_by`| String | The attribute to use for sorting
+`description`| String | An optional description of the category
+`id` | Int | An ID that uniquely identifies the category
+`level` | Int | Indicates the depth of the category within the tree
+`name`| String | The display name of the category
+`path_in_store`| String | Category path in the store
+`path`| String | The path to the category, as a string of category IDs, separated by slashes (/). For example, `1/2/20`
+`position`| Int | The position of the category relative to other categories at the same level in tree
+`product_count`| Int | The number of products in the category
+`products(<attributes>)` | `CategoryProducts` | The list of products assigned to the category
+`updated_at`| String | Timestamp indicating when the category was updated
+`url_key`| String | The url key assigned to the category
+`url_path`| String | The url path assigned to the category
+`url_suffix` | String | The part of the URL that is appended to the `url_key`, such as `.html`. This attribute is defined in the `CatalogUrlRewriteGraphQl` module
+
+### CategoryProducts object
+
+The `products` attribute can contain the following attributes:
+
+Attribute | Data type | Description
+--- | --- | ---
+`currentPage` | Int |  Specifies which page of results to return. The default value is 1.
+`pageSize` | Int | Specifies the maximum number of results to return at once. This attribute is optional. The default value is 20.
+`sort` | `ProductSortInput` | Specifies which attribute to sort on, and whether to return the results in ascending or descending order. [Searches and pagination in GraphQL]({{ page.baseurl }}/graphql/search-pagination.html) describes sort orders.
+
+The `CategoryProducts` object contains the following attributes:
+
+Attribute | Data type | Description
+--- | --- | ---
+`items` | [ProductInterface] | An array of products that are assigned to the category. See [ProductInterface]({{ page.baseurl }}/graphql/queries/products.html#ProductInterface) for more information.
```

Comment:
should we consider linking ProductInterface in the data type column rather than adding the additional sentence?